examples/structured_2d_dgsem/elixir_euler_warm_bubble.jl

```diff
@@ -0,0 +1,124 @@
+using OrdinaryDiffEq
+using Trixi
+
+# Physical constants
+g::Float64 = 9.81            # gravity of earth
+c_p::Float64 = 1004.0        # heat capacity for constant pressure (dry air)
+c_v::Float64 = 717.0         # heat capacity for constant volume (dry air)
+gamma::Float64 = c_p / c_v   # heat capacity ratio (dry air)
+
+# Warm bubble test case from
+# Wicker, L. J., and Skamarock, W. C.
+# A time-splitting scheme for the elastic equations incorporating second-order Runge–Kutta
+# time differencing
+# [DOI: 10.1175/1520-0493(1998)126%3C1992:ATSSFT%3E2.0.CO;2](https://doi.org/10.1175/1520-0493(1998)126%3C1992:ATSSFT%3E2.0.CO;2)
+function initial_condition_warm_bubble(x, t, equations::CompressibleEulerEquations2D)
+    # center of perturbation
+    center_x = 10000.0
+    center_z = 2000.0
+    # radius of perturbation
+    radius = 2000.0
+    # distance of current x to center of perturbation
+    r = sqrt((x[1] - center_x)^2 + (x[2] - center_z)^2)
+
+    # perturbation in potential temperature
+    potential_temperature_ref = 300.0
+    potential_temperature_perturbation = 0.0
+    if r <= radius
+        potential_temperature_perturbation = 2 * cospi(0.5 * r / radius)^2
+    end
+    potential_temperature = potential_temperature_ref + potential_temperature_perturbation
+
+    # Exner pressure, solves hydrostatic equation for x[2]
+    exner = 1 - g / (c_p * potential_temperature) * x[2]
+
+    # pressure
+    p_0 = 100_000.0  # reference pressure
```

Comment:
I scanned the reference in the docstring above and could not find these specific details, like the reference pressure. It also seems like you used a different (expanded) domain than the reference. Where did these values come from?

Reply:
Good point!

My first reference was 
> GEORGE H. BRYAN AND J. MICHAEL FRITSCH
> A Benchmark Simulation for Moist Nonhydrostatic Numerical Models

who refer to
> LOUIS J. WICKER AND WILLIAM C. SKAMAROCKA
> Time-Splitting Scheme for the Elastic Equations Incorporating Second-Order Runge–Kutta Time Differencing

who refer to
> CARPENTER ET AL.
> Application of the Piecewise Parabolic Method (PPM) to Meteorological Modeling Richard L.  Jr

Our implementation follows the Wicker / Skamarocka paper. In particular we adopt
- the periodic boundary conditions in horizontal direction
- the prescribed horizontal velocity
- the domain size (20 km x 10 km) and the size and position of the perturbation (it is described on page 1997)

But you are right, they do not mention the reference pressure. Neither do Carpenter et al., but Bryan / Fritsch do (page 2920). It might just be a default choice. Not ISA, but close.

Reply:
As far as I know, 10^5 Pa is a standard reference pressure, so it's fine for me. It's also used by the CLiMA group, e.g., https://github.com/mwarusz/paper-esdg-nonconservative-gravity/blob/35fbbb51824a7d696f05e2d1eac25db5207ca024/risingbubble/diagnostics_risingbubble.jl#L20

Reply:
Okay, thanks for the clarifications!